### PR TITLE
109_fix_macos_installer

### LINKED ISSUE #109: Feedback installers
Hi @paulhfu ,
this issue is for collecting feedback on the installers. 

For now I only have positive things to report:
- The CI runs through without an issue and the artifacts are uploaded to https://github.com/computational-cell-analytics/micro-sam/actions/runs/5707871215
- I tested the linux installer and it works.

@sushmita-nair and @Sagnik700: could you please test the windows installer?

@Marei33: could you please test the mac installer? 

For downloading the installers please follow the link above, and scroll to the bottom, then you can download the respective installer:

image

### PR DESCRIPTION
**Pull request by paulhfu**
Fixes #109

- do not add to system path during installation

### CODE REVIEW

**Comment by paulhfu** [comment]
Hi @constantinpape,

shall I proceed to add the signing workflow with a "Developer ID Installer certificate" for MacOs?
I created a free developer account but certificates seem to be behind a paywall.
Information about signing and the certificates can be found [here](https://help.apple.com/xcode/mac/current/#/dev033e997ca) and [here](https://developer.apple.com/help/account/create-certificates/create-developer-id-certificates/).

**Comment by constantinpape** [comment]
Hi @paulhfu ,
is the signing necessary for the installation to work, or will it only throw a warning if it's not signed?

**Comment by constantinpape** [comment]
Ah, I found it:
```
On macOS, if your app isn’t downloaded from the Mac App Store or signed with a Developer ID certificate, it won’t launch unless the user completely disables [Gatekeeper](https://help.apple.com/xcode/mac/current/#/dev190d53ccb). Users have the option of enabling or disabling identified developers in System Preferences.
```

File: deployment/construct.yaml [hunk 1]
```diff
@@ -12,4 +12,4 @@ welcome_text: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do ei
   tempor incididunt ut labore et dolore magna aliqua.
 conclusion_text: Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
   nisi ut aliquip ex ea commodo consequat.
-initialize_by_default: true
\ No newline at end of file
+initialize_by_default: false
```
Comment (constantinpape) [inline deployment/construct.yaml:15]: What does this change mean?

**Comment by constantinpape** [comment]
Ok, so from what I understand, for signing you need a developer account, which is 100$ a year. So let's not do that.
Either we:
- Use the bundled app solution (I will follow up in #109 on that)
- Or we don't provide a Mac installer

**Comment by paulhfu** [comment]
Yes, this is also how I understood it.

File: deployment/construct.yaml (hunk 1 above)
Comment (paulhfu) [reply deployment/construct.yaml:15]: This will disable the default activation of the "append to system path" - checkbox.

**Review by constantinpape — APPROVED** [review]

**Comment by constantinpape** [comment]
I will go ahead and merge it, but reopen #109 so we can discuss there if there is anything else we can do for mac.

**Pull request merged**